doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -167,38 +136,6 @@ Architecture they must be implemented as per the following specifications:
 
 <p align="center"><b>Table 4-6:</b> Storage Solution Specifications</p>
 
-<!--
-> THE BELOW TEXT HAS BEEN COMMENTED AS NEEDS REVIEWING AND REPLACED WITH SPECS IN THE ABOVE TABLE AS PER:
```

Comment:
Where are the statements in this block captured in the requirements?

Reply:
All requirements listed in this comment block are captured in ra2.stg.001, ra2.stg.002, ra2.stg.003, ra2.stg.004, ra2.stg.005, ra2.stg.006 and ra2.stg.007.